### ISSUE TITLE
Support for jruby

### ISSUE THREAD

**Issue by ndbroadbent** [gdelugre/origami#28]
There seems to be a problem when I'm using this gem with jruby. It gets tripped up by the `.to_o` method, for some reason.

```
Bundler::GemRequireError: There was an error while trying to load the gem 'origami'.
Gem Load Error is: undefined method `to_o' for 0:Fixnum
Did you mean?  to_json
               to_c
               to_r
               to_d
               to_i
               to_s
               to_f
Backtrace for gem load error is:
/Users/ndbroadbent/.rvm/gems/jruby-9.1.7.0/gems/origami-2.0.3/lib/origami/array.rb:54:in `block in initialize'
org/jruby/RubyArray.java:1733:in `each'
org/jruby/RubyEnumerable.java:1067:in `each_with_index'
/Users/ndbroadbent/.rvm/gems/jruby-9.1.7.0/gems/origami-2.0.3/lib/origami/array.rb:53:in `initialize'
/Users/ndbroadbent/.rvm/gems/jruby-9.1.7.0/gems/origami-2.0.3/lib/origami/array.rb:195:in `initialize'
/Users/ndbroadbent/.rvm/gems/jruby-9.1.7.0/gems/origami-2.0.3/lib/origami/array.rb:278:in `[]'
/Users/ndbroadbent/.rvm/gems/jruby-9.1.7.0/gems/origami-2.0.3/lib/origami/page.rb:514:in `<module:Format>'
```

JRuby version:

```bash
$ jruby --version
jruby 9.1.7.0 (2.3.1) 2017-01-11 68056ae Java HotSpot(TM) 64-Bit Server VM 25.131-b11 on 1.8.0_131-b11 +jit [darwin-x86_64]
```

**Comment by ndbroadbent** [comment]
Never mind, I think `jruby` is pretty much dying. There's no support for Rails 5.x, either.

**Comment by gdelugre** [comment]
I'm afraid this is a bug in JRuby's implementation of refinements.

Apparently it has already been reported: https://github.com/jruby/jruby/issues/4330

**Comment by ndbroadbent** [comment]
Ah thanks, that's good to know. 

A potential workaround would be a temporary branch where I remove the refinements and add them to the global classes. 

But I probably won't be using `jruby` any time soon, because `activerecord-jdbc-adapter` doesn't support Rails 5.x yet. (https://github.com/jruby/activerecord-jdbc-adapter/issues/708)

**Comment by andy-twosticks** [comment]
Note: this appears to be fixed in jruby-9.2.0.0.
